### DISCUSSION TITLE
How to disable tables and task lists when `gfm: true`?

### DISCUSSION THREAD

**Discussion by rfgamaral** [markedjs/marked#2292] [Q&A] [answered]
I need to use `gfm: true` for certain behaviour handling, but I want to completely disable support for tables and task lists that comes from this. I can't seem to find a way to do it...

Is it possible to achieve this with `marked`? Any pointers in the right direction are appreciated.

Thanks in advance.

**Comment by UziTech** [comment, marked as answer]
You can replace the table tokenizer and always return `undefined`. And for the task list the easiest thing is probably to replace the `checkbox` renderer and output the text instead of a checkbox.

Something like:

```js
const tokenizer = {
  table() {}
};

const renderer = {
  checkbox(checked) {
    return `[${checked ? 'x' : ' '}] `;
  }
};

marked.use({tokenizer, renderer});
```

**Reply by rfgamaral** [reply]
Thank you 🤗
